src/api/v1/db/oracledb/doughs-dao.js

```diff
@@ -188,19 +211,20 @@ const getDoughs = async (filters) => {
 };
 
 /**
- * Generate bind params from a POST body, assuming invalid attributes
+ * Generate bind params from a POST or PATCH body, assuming invalid attributes
  * have been removed and non-included values have been substituted for their
- * defaults
+ * defaults. If included, the return will include the values in `baseValue`.
  *
  * @param {object} body
+ * @param {object} baseValue use to include other out bind variables that are not in body
  * @returns {object} bindparams
  * @throws {Error} when an attribute not in doughsProperties is found in body.
  * It's assumed that body has already been checked for invalid properties meaning that when
  * one is found here it's a server error rather than a user error.
  */
-const getPostBindParams = (body) => {
+const getBindParams = (body, baseValue = {}) => {
   const { attributes } = body.data;
-  const bindParams = doughsOutBindParams;
+  const bindParams = _.cloneDeep(baseValue);
```

Comment:
making a shallow copy of `baseValue` meant we were modifying the original data structure, so when `postDoughs` was calling this with `doughsOutBindParams` the method was modifying `doughsOutBindParams` to also include the bind params for the original `POST` request, which would then show up in the `PATCH` request bind params, causing an error.